easybuild/easyconfigs/h/Harminv/Harminv-1.4-intel-2015a.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'Harminv'
+version = '1.4'
+
+homepage = 'http://ab-initio.mit.edu/wiki/index.php/Harminv'
+description = """Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion - given
+ a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially decaying)
+ in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those sinusoids."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'opt': True, 'unroll': True, 'optarch': True, 'pic': True, 'cstd': 'c99'}
+
+source_urls = ['http://ab-initio.mit.edu/harminv/']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = "--with-pic  --with-blas=mkl_em64t --with-lapack=mkl_em64t --enable-shared"
```

Comment:
from the configure output:

```
checking for sgemm_ in -lmkl_em64t... yes                                                                                                                                                                                               checking for cheev_ in -lmkl_em64t... yes  
```

This works because of https://github.com/hpcugent/easybuild-easyblocks/blob/master/easybuild/easyblocks/i/imkl.py#L159